app/src/controllers/sorting/withSortingURL.jsx

```diff
@@ -39,6 +39,8 @@ export const withSortingURL = ({
       const fields = oldFields.length > 0 ? oldFields : defaultFields;
       if (fields.includes(field)) {
         direction = direction === SORTING_ASC ? SORTING_DESC : SORTING_ASC;
+      } else {
+        direction = oldDirection ? SORTING_ASC : defaultDirection;
```

Comment:
I think there should be `direction = defaultDirection;`

Reply:
No. I discussed this with BA and she approved current behavior.